_docker/drupal/drupal-filesystem/web/modules/custom/redhat_developers/src/Twig/RedHatDevelopersTagsExtension.php

```diff
@@ -7,23 +7,25 @@
 
 namespace Drupal\redhat_developers\Twig;
```

Comment:
Yeah, though I'm not sure how to set it up. I know @drupalbee is working on that. I can give it a go, though I'm not quite sure how we hook it up into the current pipeline.